recipes/parallel-preprocessor/meta.yaml

```diff
@@ -0,0 +1,80 @@
+# manually update or generated from template
+package:
+    name: parallel-preprocessor
+    version: '0.3'
+source:
+    # calc check sum by command sha256sum
+    sha256: 0f472735203b16e092686003e50e25d2e6dbdeef43da8f14177d6bcb4db86634
+    url: https://github.com/ukaea/parallel-preprocessor/releases/download/v0.3/parallel-preprocessor.zip
+
+build:
```

Comment:
As this primarily provides a shared library, should it use `run_exports` to ensure downstream packages are pinned appropriately?

Reply:
parallel-preprocessor is a end-user tool, not expected to be used as a lib in 5 years.                  ah.  I do not know  "run_exports"